x/valset-pref/client/cli/tx.go

```diff
@@ -2,34 +2,52 @@ package valsetprefcli
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/osmosis-labs/osmosis/osmoutils"
 	"github.com/osmosis-labs/osmosis/osmoutils/osmocli"
 	"github.com/osmosis-labs/osmosis/v13/x/valset-pref/types"
 )
 
 func GetTxCmd() *cobra.Command {
 	txCmd := osmocli.TxIndexCmd(types.ModuleName)
```

Comment:
Can we just change this to directly use the string valset-pref instead of utilizing validatorsetpreference for every command?

Reply:
aah so change from `osmosisd tx validatorsetpreference...` to `osmosisd tx valset-pref` ?

Reply:
its not a blocker or anything, we can leave as is